focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -432,12 +432,18 @@ class BrowserFragment :
         }
     }
 
-    override fun onAccessibilityStateChanged(enabled: Boolean) = when (enabled) {
-        false -> binding.browserToolbar.enableDynamicBehavior(requireContext(), binding.engineView)
-        true -> {
-            with(binding.browserToolbar) {
-                disableDynamicBehavior(binding.engineView)
-                showAsFixed(requireContext(), binding.engineView)
+    override fun onAccessibilityStateChanged(enabled: Boolean) {
+        when (enabled) {
+            // using _binding, because this might be called before onCreateView.
+            false -> _binding?.browserToolbar?.enableDynamicBehavior(
+                requireContext(),
+                binding.engineView,
```

Comment:
should the other references to binding in this block use _binding?

Reply:
It's technically the same thing, if `_binding` is not null, `binding` is also not null, so there is no change of an NPE.

Using `_binding` for calling `enableDynamicBehavior` would need a non-null asserted call or another let block.
I preferred to keep it in the original format. Let me know if you feel strongly otherwise.

Reply:
Understood 👍 ok, lgtm ✅